From c8c0b7f49b5d3c897a987d78772dd7e3edaa8e91 Mon Sep 17 00:00:00 2001
From: Hans de Graaff <hans@degraaff.org>
Date: Sat, 14 Jul 2018 08:59:26 +0200
Subject: [PATCH] Fix syntax error in make install sh code

if should be closed with fi, not done.

Fixes:

if [ "linux" = "hp-ux" ]; then mv /var/tmp/portage/net-analyzer/munin-2.0.38/image//usr/libexec/munin/plugins/*.adv /var/tmp/portage/net-analyzer/munin-2.0.38/image//usr/libexec/munin; done
/bin/sh: -c: line 0: syntax error near unexpected token `done'
/bin/sh: -c: line 0: `if [ "linux" = "hp-ux" ]; then mv /var/tmp/portage/net-analyzer/munin-2.0.38/image//usr/libexec/munin/plugins/*.adv /var/tmp/portage/net-analyzer/munin-2.0.38/image//usr/libexec/munin; done'
make: *** [Makefile:146: install-plugins-prime] Error 1
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5bfb914a5..48c7708f0 100644
--- a/Makefile
+++ b/Makefile
@@ -149,7 +149,7 @@ install-plugins-prime: install-plugins build $(PLUGINS) Makefile Makefile.config
 	    fi                                                 \
 	done
 	@# Some HP-UX plugins need *.adv support files in LIBDIR
-	if [ "$(OSTYPE)" = "hp-ux" ]; then mv $(LIBDIR)/plugins/*.adv $(LIBDIR); done
+	if [ "$(OSTYPE)" = "hp-ux" ]; then mv $(LIBDIR)/plugins/*.adv $(LIBDIR); fi
 	$(INSTALL) -m 0644 build/plugins/plugins.history $(LIBDIR)/plugins/
 	$(INSTALL) -m 0644 build/plugins/plugin.sh $(LIBDIR)/plugins/
 
